demo5/ReadMe.txt

```diff
@@ -0,0 +1,23 @@
+Demonstration of using message queues together with the `selectors` module in Python.
```

Comment:
Feel free to change this to a `.md` file, especially since it already contains markdown. Eventually I'd like to change all of my doc to Markdown.